### DISCUSSION TITLE
Plot style configurability & declarative plotting comments

### DISCUSSION THREAD

**Discussion by miptalan** [kdbinsidebrains/plugin#50] [Ideas]
First, I really like your plugin - as it feel much more “intergrated” than others.

I’ve been using qpad and kdb studio mostly, so want to share couple ideas on convenient functionalities for these apps.

 Kdb+ studio plugin has couple cool features around plotting:
- simple json-based plugin-level file configs for plot styles - allowing to set colors, lines styles, all the standard stuff
- (quite innovative) declarative plot definition _as comments_ in front of q statement returning tabular data, i.e. comment containin description of which columns to plot, axis labels, title, etc. So when executed - plot appears as a result.

Would you consider implementing such functionality?

**Comment by smklimenko** [comment]
sorry but I don't need that at this moment so not in near future

**Comment by smklimenko** [comment]
Long time ago Charting Templates were introduced to cover the same: https://www.kdbinsidebrains.dev/features/charting#templates
